workspaces/marketplace/plugins/marketplace-common/src/types/MarketplaceSupport.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @public
+ */
+export enum SupportLevel {
+  PRODUCTION = 'production',
+  TECH_PREVIEW = 'tech-preview',
+  DEV_PREVIEW = 'dev-preview',
+}
+
+/**
+ * @public
+ */
+export enum SupportProvider {
+  RED_HAT = 'Red Hat',
+  BACKSTAGE_COMMUNITY = 'Backstage Community',
+  VENDOR = 'Vendor',
+}
```

Comment:
The support provider should be just a string. I like to avoid mention Red Hat here as long as possible to allow us moving this to Backstage Community Plugins later.